runhouse/main.py

```diff
@@ -666,6 +341,96 @@ def cluster_list(
         print_sky_clusters_msg(len(sky_clusters))
 
 
+@cluster_app.command("keep_warm")
+def keep_warm(
```

Comment:
can we add a docstring here so it also appears in the "help" command: 
"Keep the cluster warm for given number of minutes after inactivity"